docs/resources/as_group.md

```diff
@@ -223,3 +223,9 @@ This resource provides the following timeouts configuration options:
 - `create` - Default is 10 minute.
 - `delete` - Default is 10 minute.
 
+## Import
+
+AS groups can be imported by their `id`. For example,
+```
+terraform import huaweicloud_as_group.import_as 9ec5bea6-a728-4082-8109-5a7dc5c7af74
```

Comment:
it's better to name the resource as `my_as_group`

Reply:
fixed